src/components/Table/Table.tsx

```diff
@@ -8,7 +8,7 @@ export interface Props {
    * - **overflow-sm** Removes the min-width from the table for tables with few columns
    * - **overflow-lg** Used for tables with lots of table rows to overflow at a larger breakpoint
    */
-  behavior?: 'overflow';
+  behavior?: 'overflow' | 'overflow-lg' | 'overflow-sm';
```

Comment:
I changed this because of a TS warning I saw when I opened this file; it doesn't relate to the import work in this PR and can be moved to a separate PR if requested.

Reply:
Nice, thanks!